source/userguide/active_learning.rst

```diff
@@ -0,0 +1,226 @@
+| save_sampled_queries   | bool       | Whether to save the queries sampled at each iteration                            |
++------------------------+------------+----------------------------------------------------------------------------------+
+| aggregate_statistic    | str        | Metric to record aggregate classifier performance data in the output and plots,  |
+|                        |            | options: "accuracy", "f1_weighted", "f1_macro", "f1_micro"                       |
++------------------------+------------+----------------------------------------------------------------------------------+
+| class_level_statistic  | str        | Metric to record class/tuning level performance data in the output and plots,    |
+|                        |            | options: "f_beta", "percision", "recall"                                         |
++------------------------+------------+----------------------------------------------------------------------------------+
+
+5. **query_selection**
+
++------------------------+------------+----------------------------------------------------------------------------------+
+| Configuration Key      | Type       | Definition and Examples                                                          |
++========================+============+==================================================================================+
+| selection_strategy     | str        | Strategy (heuristic) to use for log selection                                    |
++------------------------+------------+----------------------------------------------------------------------------------+
+| log_usage_pct          | float      | Percentage of the log data to use for selection                                  |
++------------------------+------------+----------------------------------------------------------------------------------+
+| labeled_logs_pattern   | str        | Regex pattern to match log files if already labeled. For example, ".*log.*.txt"  |
++------------------------+------------+----------------------------------------------------------------------------------+
+| unlabeled_logs_path    | str        | Path to text file containing unlabeled queries from user logs or other resources |
++------------------------+------------+----------------------------------------------------------------------------------+
+
+
+If there is no configuration defined in the ``config.py`` file or if fields are missing in the custom configuration, the relevant missing information is obtained from the following default configuration:
+
+.. code-block:: python
+
+    DEFAULT_ACTIVE_LEARNING_CONFIG = {
+        "output_folder": None,
+        "pre_tuning": {
+            "train_pattern": ".*train.*.txt",
+            "test_pattern": ".*test.*.txt",
+            "train_seed_pct": 0.20,
+        },
+        "tuning": {
+            "n_classifiers": 3,
+            "n_epochs": 5,
+            "batch_size": 100,
```

Comment:
Would be good to add a warning or note somewhere that the default batch size is 100. For large applications, this number may be too small and we should encourage developers to update it accordingly. (Recommend 1-2% of total training data size, which is what most papers seem to do)